Assets/MixedRealityToolkit/_Core/Utilities/UX/HoverLight.cs

```diff
@@ -51,28 +56,31 @@ public Color Color
 
         private void OnEnable()
         {
-            Initialize();
+            AddHoverLight(this);
         }
 
         private void OnDisable()
         {
-            UpdateHoverLight();
+            RemoveHoverLight(this);
+            UpdateHoverLights(true);
         }
 
+#if UNITY_EDITOR
```

Comment:
Why did this get reverted?

Reply:
This is based on the changes that went into Dev_Working_Branch and back into june18_dev. Part of the gap analysis / keeping things in sync. If this change is incorrect, we should definitely fix it here and in june18_dev.

Reply:
Please remove compiler directive in favor of checking for `Application.isEditor`